### ISSUE TITLE
[Bug]: Accordions don't fully announce state

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shinyGovstyle#191]
### Overview

Users will encounter accordions beginning at ‘Show all sections’ and ending at ‘How people read’. Encountering these accordions with screen-reading software, a screen reader user will not hear a collapsed or expanded state. The software is unable to announce a state because the aria-expanded attribute is omitted. This means that users will not know the current state.

### Steps to Reproduce

Inspect the HTML on the showcase app.

### Expected Behavior

`aria-expanded="false"` and `aria-expanded="true"` should be added to each accordion within the button HTML, e.g.

```
<button aria-expanded="false" class="govuk-accordion__section-button" 
id="accordion-default-heading-01" name="accordion-default-heading-01">
 <span class="govuk-accordion__section-heading-text">
 <span class="govuk-accordion__section-heading-text-focus">Writing well for the web</span>
 </span>
 <span class="govuk-visually-hidden govuk-accordion__section-heading-divider"></span>
 <span class="govuk-accordion__section-toggle">
 <span class="govuk-accordion__section-toggle-focus">
 <span class="govuk-accordion-nav__chevron govuk-accordion-nav__chevron--
down"></span>
 <span class="govuk-accordion__section-toggle-text">
 Show
 <span class="govuk-visually-hidden">this section</span>
 </span>
 </span>
 </span>
</button>

```

### Additional Information

Page 34 of the September 2025 accessibility report
